### ISSUE TITLE
installation fails on Ubuntu 18.04 due do dependency on ancient json vesion

### ISSUE THREAD

**Issue by matkoniecz** [backup/backup#944]
## What went wrong?

Installation failed during test install of 18.04 Ubuntu
see https://gist.github.com/matkoniecz/3e23d7972f9e8d73acbe2678fff9d264
it appears that it depends on json version from 2015 - see https://rubygems.org/gems/json/versions/1.8.2

## What steps did you follow?

1. sudo gem install backup
1. sudo gem install backup --version 4.3.0 (as it was in template of issue)

## How is your copy of backup configured?

not relevant

## Tell us about the computer that runs the backup gem

`uname -a` gives `Linux golem 4.18.0-25-generic #26~18.04.1-Ubuntu SMP Thu Jun 27 07:28:31 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux`

* Operating system: Lubuntu 18.04
* Ruby version: 2.5.1p57
* Tools or services used by backup: None

**Comment by piotr-galas** [comment]
I think it is because of json 1.8.2 is incompatibile with ruby 2.4.0

See this issue https://github.com/flori/json/issues/303

I encounter the same problem. I do not know how to fix it yet.

**Comment by piotr-galas** [comment]
https://github.com/backup/backup/issues/877

TLDR;
`gem install backup -v5.0.0.beta.1`

**Comment by matkoniecz** [comment]
Closing as technically it is solved - though this version will be never released as this project is dead.

Maybe I will submit this unreleased version as a separate gem, and potentially maintain parts that I use?

**Comment by matkoniecz** [comment]
I am tempted to open a new issue "Release 5.0.0"
